homeassistant/components/freebox/sensor.py

```diff
@@ -155,8 +155,6 @@ def update(self) -> None:
     def device_state_attributes(self) -> Dict[str, any]:
         """Return device specific state attributes."""
         return {
-            datetime.fromtimestamp(call["datetime"], TIMEZONE).strftime(
-                "%d/%m/%Y, %H:%M:%S"
-            ): call["name"]
+            datetime.fromtimestamp(call["datetime"], TIMEZONE).isoformat(): call["name"]
```

Comment:
Please use this util:
https://github.com/home-assistant/core/blob/8895f9b70a3f93a29a573847f0f9370936506700/homeassistant/util/dt.py#L97-L99

Reply:
With `dt_util.utc_from_timestamp(call["datetime"]).isoformat(): call["name"]`, my last missed phone call is at `2020-06-19T08:48:53+00:00`.

With `datetime.fromtimestamp(call["datetime"], TIMEZONE).isoformat(): call["name"]`, my last missed phone call is at `2020-06-19T10:48:53+02:00`

With the Freebox app for my last missed phone call is `19/06/2020 at 10h48`.

--

Can I use `utc_from_timestamp` with a timezone ?
Otherwise displayed data will be wrong of 1 (winter time) ou 2 hours (summer time).

Reply:
What timezone is 10:48? All times should be UTC time in the state machine.

Reply:
Europe/Paris

The Freebox is **NOT** giving us UTC timestamp bot French localised timestamp.

Reply:
Isn't 8:48 in UTC the same as 10:48 in CEST?

Reply:
> Central European Summer Time (CEST) is 2 hours ahead of Coordinated Universal Time (UTC).

https://www.timeanddate.com/time/zones/cest

Reply:
Yes, exactly. So then 8.48 in UTC is correct since we want the time reported in UTC time.

Reply:
Timestamps should be stored as UTC and be teh state of the sensor. You can then use the `DEVICE_CLASS_TIMESTAMP` and allow the user to choose how to display it in the frontend (relative, absolute etc) customized to their locale.

Reply:
In UTC now.

Indeed I was wrong about the localised timestamp apparently, datetime stuffs are still not my cup of tea :sweat_smile: 

Result:
![image](https://user-images.githubusercontent.com/14821482/85895218-8a660100-b7f6-11ea-9185-1260a196fe11.png)


---

Yep @balloob but this sensor is a number of missed calls, not the date of the last missed call, we were talking about the attributes :wink: 
But it's a good idea, will see if people are interested in that sensor.